olexandr.vakulinskiy/HW2/Task1/tamagotchi.rb

```diff
@@ -0,0 +1,113 @@
+class Tamagotchi
+  attr_accessor :name, :health, :hunger, :thirst, :fatigue
+
+  def initialize(name)
+    @name = name
+    @health = 10
+    @hunger = 2
+    @thirst = 2
+    @fatigue = 0
+  end
+
+  def check_health
+    if @hunger.zero? || @thirst.zero?
+      @health += 1
+    elsif @hunger >= 7 || @thirst >= 7
+      @health -= 1
+    end
+
+    case @fatigue
+    when 10
+      @health = 0
+    when 7...10
+      @health -= 2
+    end
+  end
+
+  def feed
+    check_health
+    @hunger -= 1
+    puts 'You fed your pet.' unless @health == 0
+  end
+
+  def drink
+    check_health
+    @thirst -= 1
+    puts 'You gave some water to your pet.' unless @health == 0
+  end
+
+  def play
+    check_health
+    @fatigue += 2
+    @hunger += 1
+    @thirst += 1
+    puts 'You played different games with your pet.' unless @health == 0
+  end
+
+  def sleep
+    check_health
+    @fatigue -= 1
+    @hunger += 2
+    @thirst += 2
+    puts 'Your pet is sleeping now.' unless @health == 0
+  end
+
+  def dead?
+    true ? @health <= 0 : false
```

Comment:
This condition won't work. Change `true`

Reply:
What do you mean?

Reply:
@AlexanderVak This condition `true ? @health <= 0 : false` will always work, because `true` - is always a positive condition that does not change. What is the point of such checking?